tests/service/privacy_request/request_runner_service_test.py

```diff
@@ -240,6 +260,58 @@ def test_create_and_process_erasure_request_generic_category(
     assert customer_found
 
 
+@pytest.mark.integration_erasure
+def test_create_and_process_erasure_request_aes_generic_category(
+        postgres_example_test_dataset_config,
+        cache,
+        db,
+        generate_auth_header,
+        erasure_policy_aes,
+        connection_config,
+):
+    # It's safe to change this here since the `erasure_policy` fixture is scoped
+    # at function level
+    target = erasure_policy_aes.rules[0].targets[0]
+    target.data_category = DataCategory("user.provided.identifiable.contact").value
+    target.save(db=db)
+
+    email = "customer-2@example.com"
+    customer_id = 2
+    data = {
+        "requested_at": "2021-08-30T16:09:37.359Z",
+        "policy_key": erasure_policy_aes.key,
+        "identity": {"email": email},
+    }
+
+    pr = get_privacy_request_results(db, erasure_policy_aes, cache, data)
+    pr.delete(db=db)
+
+    example_postgres_uri = PostgreSQLConnector(connection_config).build_uri()
+    engine = get_db_engine(database_uri=example_postgres_uri)
+    SessionLocal = get_db_session(engine=engine)
+    integration_db = SessionLocal()
+    stmt = select(
+        column("id"),
+        column("email"),
+        column("name"),
+    ).select_from(table("customer"))
+    res = integration_db.execute(stmt).all()
+
+    customer_found = False
+    for row in res:
+        if customer_id in row:
+            customer_found = True
+            # Check that the `email` field is `None` and that its data category
+            # ("user.provided.identifiable.contact.email") has been erased by the parent
+            # category ("user.provided.identifiable.contact")
+            assert row[1] is not "customer-2@example.com"
+            assert row[2] is not None
```

Comment:
Just tested this, it works, thanks @eastandwestwind! One last small ask: can we test for the encrypted value here too?